toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -108,5 +109,14 @@ func (s *SystemConfig) IsValid() error {
 		}
 	}
 
+	if s.OverlayFS != nil {
+		for i, overlayFS := range *s.OverlayFS {
```

Comment:
You should additionally check if upper and works dirs are unique among the overlays. Lower dirs may be shared, but upper and work dirs must be unique.

Reply:
Fixed.